core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -238,6 +238,7 @@ public String toString() {
   private final long currentSnapshotId;
```

Comment:
Do we need to handle all the update snapshot calls?  ie, replaceCurrentSnapshot, addStagedSnashot, etc.